### PR TITLE
Task/add grants doc

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: docs/authorization_api.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+Authorization API allows you to create, assign and retrieve grants for users of Thinking Cities platform.
+
+# Grants
+
+Grants are a kind of subservices (keystone projects) which can implies for an user that has any role in that subservice a permission for do something in an apllication built on top of ThingkinCities Platform.
```
Comment (fgalan) [inline docs/authorization_api.md:5]:
```suggestion
Grants are a kind of subservices (keystone projects) which can imply for a user that has any role in that subservice a permission for do something in an application built on top of ThingkinCities Platform.
```

File: docs/authorization_api.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+Grants always starts with a '#' characters, in contrast to others subservices which starts without that character.
```
Comment (fgalan) [inline docs/authorization_api.md:8]:
```suggestion
Grants always start with a '#' character, in contrast to other subservices which start without that character.
```

File: docs/authorization_api.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Create Grants
+
+Grants, like other subservices (keystone project), can be created using [Management API for create subservice](https://thinking-cities.readthedocs.io/en/latest/management_api/index.html#create-subservice)
+
+Morever, like others subservices, can be created using Administrationt Portal of ThinkingCities Platform.
+
+
+# Assign Grants
+
+For assign a grant to a user a role should be asigned to a user in that grant (keystone project) using [Management API for assign role to user](https://orchestrator2.docs.apiary.io/#reference/orchestrator/user-role-assigment/assign-role-to-user)
```
Comment (fgalan) [inline docs/authorization_api.md:24]:
```suggestion
To assign a grant to a user, a role should be assigned to a user in that grant (keystone project) using [Management API for assign role to user](https://orchestrator2.docs.apiary.io/#reference/orchestrator/user-role-assigment/assign-role-to-user)
```

File: docs/authorization_api.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Morever, like others role assigments, can be performed using Administrationt Portal of ThinkingCities Platform.
```
Comment (fgalan) [inline docs/authorization_api.md:26]:
```suggestion
Morever, like others role assigments, can be performed using Administration Portal of ThinkingCities Platform.
```

File: docs/index.md [hunk 5]
```diff
@@ -18,6 +18,7 @@ These are the main benefits of Thinking Cities platform:
 Thinking Cities provides the following APIs:
 
 - [Authentication API](authentication_api.md): manages tokens for APIs usage.
+- [Authorization with grants API](authorization_api.md): manages grants.
```
Comment (fgalan) [inline docs/index.md:21]: Maybe it should be added also to mkdocs.yml so the file also appears in the RTD rendering of the doc at http://thinking-cities.readthedocs.io/en/master/

Comment (AlvaroVega) [reply docs/index.md:21]: done in https://github.com/telefonicaid/thinking-cities/pull/130/commits/433f12ba35f6b2c426d4735f99a98c1e45a4fd9e

**Review by fgalan — APPROVED** [review]
LGTM

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**